index.js

```diff
@@ -293,33 +288,39 @@ client.on(Events.InteractionCreate, async (interaction) => {
                 const messageTracker = await generalChat.messages.fetch(MESSAGE_TRACKER_ID);
 
                 // Grab current message tracker reviews 
-                const currentMessageTrackerContent = messageTracker.content;
+                const messageTrackerContent = messageTracker.content;
 
                 // Check if reviewer already has reviews in message tracker
-                const userIsAlreadyReviewer = checkForMatchInMessage(currentMessageTrackerContent, reviewerId);
+                const userIsAlreadyReviewer = checkForMatchInMessage(messageTrackerContent, reviewerId);
 
                 // Split current message tracker reviews line by line
-                const currentMessageTrackerContentByLineArray = currentMessageTrackerContent.split(LINE_REGEX);
+                const messageTrackerContentByLineArray = messageTrackerContent.split(LINE_REGEX);
+
+                // Define non-default value for adding new reviewers and reviews
+                const isFirstBlock = false;
 
                 // New reviewer added to message tracker 
                 if (!userIsAlreadyReviewer) {
-                    getPositionToAddReviewer(currentMessageTrackerContentByLineArray, review, reviewerMention);
-                    messageTracker.edit(currentMessageTrackerContentByLineArray.join('\n'));
+                    const indexToAddReviewAndReviewer = getPositionToAddReview(messageTrackerContentByLineArray);
+                    addReviewAndReviewer(messageTrackerContentByLineArray, indexToAddReviewAndReviewer, review, reviewerMention)
+                    messageTracker.edit(messageTrackerContentByLineArray.join('\n'));
                     interaction.reply('Review added!');
                     return;
                 }
 
                 // Reviewer is second block
-                if (currentMessageTrackerContentByLineArray[1] !== reviewerMention) {
-                    getPositionToAddReviewTwo(currentMessageTrackerContentByLineArray, review);
-                    messageTracker.edit(currentMessageTrackerContentByLineArray.join('\n'));
+                if (messageTrackerContentByLineArray[1] !== reviewerMention) {
+                    const indexToAddReview = getPositionToAddReview(messageTrackerContentByLineArray, isFirstBlock);
```

Comment:
Slight optimization: I see `firstBlock` is a constant, so it's always one value. What's stopping us from replacing this with just `false` here at the end?